src/coreclr/jit/lsra.h

```diff
@@ -1416,7 +1416,7 @@ class LinearScan : public LinearScanInterface
         }
         return nextConsecutiveRefPositionMap;
     }
-    FORCEINLINE RefPosition* getNextConsecutiveRefPosition(RefPosition* refPosition);
+    RefPosition* getNextConsecutiveRefPosition(RefPosition* refPosition);
```

Comment:
This change should be reverted. I was trying out something for which I had to remove the `FORCEINLINE`, but I didn't made that change and hence no need to change this.